psscriptanalyzer/Initialize-Environment.ps1

```
@@ -0,0 +1,60 @@
+    param([string]$TemplateDir)
+    cd $TemplateDir
+    ls *.tmpl | % {
+        $rendered = Invoke-Expression "Get-Content $($_.FullName) | Render-Template"
+        $outputName = $_.Name -replace '\.tmpl$', ''
+        Set-Content -Path $outputName -Value $rendered
+    }
+}
+
+function Get-EnvironmentVariables {
+    $vars = Get-ChildItem env:
+    $unused = "this var is never read"
```

PSScriptAnalyzer (1.25.0)

[warning] The variable 'unused' is assigned but never used.

(PSUseDeclaredVarsMoreThanAssignments)